src/js/slider.js

```diff
@@ -0,0 +1,150 @@
+import { changeSelectedMarkers } from "./filter";
+
+const thumbsize = 14;
+const stringIntervals = [
+  "100",
+  "500",
+  "1k",
+  "5k",
+  "10k",
+  "50k",
+  "100k",
+  "500k",
+  "1M",
+  "5M",
+  "10M",
+  "50M",
+];
+
+const numInterval = [
+  100, 500, 1000, 5000, 10000, 50000, 100000, 500000, 1000000, 5000000,
+  10000000, 50000000,
+];
+
+const draw = (slider, low, high) => {
+  // Setting vars from top to bottom.
+  var lower = document.querySelector(".lower");
```

Comment:
We want to use `const` for each of these. `var` is the old API and has a lot of weird edge cases.